packages/backend/src/database/migrations/20250807212731_add_custom_roles.ts

```diff
@@ -0,0 +1,184 @@
+import { Knex } from 'knex';
+import { type DbScopeInsert } from '../entities/roles';
+
+const ScopesTableName = 'scopes';
+const RolesTableName = 'roles';
+const ScopedRolesTableName = 'scoped_roles';
+
+export async function up(knex: Knex): Promise<void> {
+    // Create roles table
+    await knex.schema.createTable(RolesTableName, (table) => {
+        table
+            .uuid('role_uuid')
+            .primary()
+            .defaultTo(knex.raw('uuid_generate_v4()'));
+        table.string('name', 100).notNullable();
```

Comment:
why 100 ? is 255 the default ?

Reply:
That's right. I suppose it can simply be the default. No strong reason to limit the length